frontend/src/pages/Common/DatePicker.js

```diff
@@ -26,6 +26,7 @@ const styles = {
 function DatePicker({ classes, name, label, onChange, onDelete, datetime, id = "default" }) {
   const dateFormat = strings.format.dateFormat;
   const datePlaceholder = strings.format.datePlaceholder;
+  const dateValue = datetime ? datetime : null;
   return (
```

Comment:
a better solution in my opinion, because if the string is empty, your version will not work. 
This will set dateValue to null if dateTime is `"", undefined, null, {}, []`:

`const dateValue = _isEmpty(datetime) ? datetime : null;`

Reply:
`import _isEmpty from "lodash/isEmpty";`